content/en/docs/Reference/auroraboot.md

```diff
@@ -386,6 +389,7 @@ cloud_config: |
 | `cloud_config`        | Cloud config path to use for the machines. A URL can be specified, use `-` to pass-by the cloud-config from _STDIN_                                                                                                                                               |
 | `iso.data`            | Defines a path to be embedded into the resulting iso. When booting, the files will be accessible at `/run/initramfs/live`                                                                                                                                         |
 | `netboot.cmdline`     | Override the automatically generated cmdline with a custom one to use during netboot. `config_url` and `rootfs`  are automatically constructed. A reasonable value can be `netboot.cmdline=rd.neednet=1 ip=dhcp rd.cos.disable netboot install-mode console=tty0` |
+| `disk.state_size`     | Set the minimum size (in MB) for the state partition in raw disk images. By default, the state partition is sized to 3 times the size of the current image plus some additional space for system files. Use this option to override the default size if you need to accommodate larger future images. |
```

Comment:
can we add `disk.size` here as well? It marks the final size image that will be generated. By default it uses the smallest possible size based on the actual data, but maybe its useful to let people now that if you set `disk.size=16000` you will get a final image of 16Gb

Reply:
there is also no mention of `disk.efi` or `disk.bios` here, should we add it?

Reply:
added disk.{efi|size|bios|vhd|gce}